content-services/content-sources-backend
# Fixes 1501: add API to list snapshots

## Summary
Adds an API endpoint to list the snapshots of a repository: `repositories/:uuid/snapshots/`
## Testing steps
1. Using the API, create a repository, and in the body include the field `snapshot: true`
2. GET request `repositories/{new repository UUID}/snapshots/`
3. Should see the snapshot returned, including pagination metadata.
